sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -111,3 +227,43 @@ pub struct ReadDatabaseOptions<'a> {
 pub struct ThroughputOptions<'a> {
     pub method_options: ClientMethodOptions<'a>,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn item_options_as_headers() {
```

Comment:
Let's add a test with everything set to `None` (which you can probably do by just creating `ItemOptions::default()`) and verify we get an empty vector back.